mautic/mautic
# Tell the user when files exceed upload_max_filesize

Please answer the following questions. 

| Q  | A
| --- | ---
| Bug fix? | No
| New feature? | Yes
| Related user documentation PR URL | N/A
| Related developer documentation PR URL | N/A
| Issues addressed (#s or URLs) | N/A
| BC breaks? | No
| Deprecations? | No

#### Description:

When importing contacts, if the size of the CSV file is too large, the user will get a generic error message not letting them know what's wrong. This pull request makes this error message more specific

#### Steps to test this PR:

1. Import contacts using a CSV file which is too large (usually above 2MB)
2. See the error message
